Group_Facilitator_Name.ipynb

```diff
@@ -1083,7 +1080,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "quantitative_cols=['loan_amnt','dti','funded_amnt','installment','funded_amnt_inv','int_rate','annual_inc','grade']\n",
```

Comment:
why did you remove this column?

Reply:
We cannot do sns.boxplot(x='grade', data=data, ax=p[1]) as grade is a categorical variable. While calling univariate_analysis(), we should not pass 'grade`